pwa-shell/src/ui/content-list/CategoryList.js

```diff
@@ -13,35 +14,38 @@ class CategoryList extends PureComponent {
 
     return categories.length ? (
       <div className="CategoryList">
-        {
-          selectedCategoryCodes.length
-            ? (
-              <div className="CategoryList__header">
-                <FilterIcon />
-                <span className="CategoryList__title">Argomenti che stai visualizzando</span>
-              </div>
-            )
-            : ''
-        }
-        {
-        categories.filter(category => selectedCategoryCodes.includes(category.code)).map(category => (
-          <span className="CategoryList__item"
-            key={category.code}
-          >
-            <Badge>{ category.titles['it'] }</Badge>
-          </span>
-        ))
-      }
+        {selectedCategoryCodes.length ? (
+          <div className="CategoryList__header">
+            <FilterIcon />
+            <span className="CategoryList__title">
+              <FormattedMessage
+                id="categorylist.topicsViewing"
+                defaultMessage="Topics you are viewing"
+              />
+            </span>
+          </div>
+        ) : null}
+        {categories
+          .filter(category => selectedCategoryCodes.includes(category.code))
+          .map(category => (
+            <span className="CategoryList__item" key={category.code}>
+              <Badge>{category.titles['it']}</Badge>
```

Comment:
You should pass the current locale id and no more the hardcoded `it`.
There are other occurrences of this titles['it'] in the codebase as well.